src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/SetNoteStatusRequestHandlerTests.cs

```diff
@@ -85,6 +85,56 @@ public async Task ExternalUser_CanSetApprovedStatus()
             note.Status.Should().Be(NoteStatus.Approved);
         }
 
+        [Fact]
+        public async Task ExternalUser_CanSetRejectedStatus()
```

Comment:
could you update the name of the test to have **HandleAsync**_ExternalUser_CanSetRejectedStatus

Reply:
method renamed